src/pkg/message/credentials.go

```diff
@@ -136,6 +137,14 @@ func PrintCredentialUpdates(oldState *types.ZarfState, newState *types.ZarfState
 			pterm.Printfln("    %s: %s", pterm.Bold.Sprint("URL Address"), compareStrings(oA.Address, nA.Address, false))
 			pterm.Printfln("    %s: %s", pterm.Bold.Sprint("Push Username"), compareStrings(oA.PushUsername, nA.PushUsername, false))
 			pterm.Printfln("    %s: %s", pterm.Bold.Sprint("Push Token"), compareStrings(oA.PushToken, nA.PushToken, true))
+		case AgentKey:
+			oK := oldState.AgentTLS
```

Comment:
nit: I don't like calling variables variations of `ok` that are not bool as it is harder to understand behavior at first glance

Reply:
https://www.youtube.com/watch?v=yGUNrHWZaVs

Reply:
changed to oT / nT instead.